clang/test/CIR/CodeGen/globals.c

```diff
@@ -0,0 +1,9 @@
+// There seems to be some differences in how constant expressions are evaluated
+// in C vs C++. This causees the code gen for C initialized globals to be a
+// bit different from the C++ version. This test ensures that these differences
+// are accounted for.
+
+// RUN: %clang_cc1 -emit-cir %s -o - | FileCheck %s
```

Comment:
Please always include the `-triple x86_64-unknown-linux-gnu` in tests. We currently run CI that cross-compiles to `x86_64-apple-darwin` and thus fails since we don't have that target supported in `CIRGenModule::getTargetCIRGenInfo`.

Reply:
BTW once you fix this it's good to land, you don't have to resubmit again to get approval.